Collector/StackPlugin.php

```diff
@@ -49,12 +49,22 @@ public function __construct(Collector $collector, Formatter $formatter, $client)
      */
     public function handleRequest(RequestInterface $request, callable $next, callable $first)
     {
-        $stack = new Stack($this->client, $this->formatter->formatRequest($request));
+        $stack = new Stack(
+            $this->client,
+            $request->getMethod(),
+            $request->getRequestTarget(),
+            $this->formatter->formatRequest($request)
+        );
 
         $this->collector->addStack($stack);
 
         return $next($request)->then(function (ResponseInterface $response) use ($stack) {
             $stack->setResponse($this->formatter->formatResponse($response));
+            $stack->setResponseCode($response->getStatusCode());
+
+            if ($response->hasHeader('X-Duration')) {
```

Comment:
ah, now i got it, this depends on https://github.com/php-http/stopwatch-plugin/pull/3